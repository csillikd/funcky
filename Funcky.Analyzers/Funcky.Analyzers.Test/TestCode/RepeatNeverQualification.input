namespace WithUsingStatic
{
    using static System.Linq.Enumerable;

    public class C
    {
        public void M()
        {
            _ = Repeat("value", 0);
        }
    }
}

namespace WithFullyQualifiedName
{
    public class C
    {
        public void M()
        {
            _ = System.Linq.Enumerable.Repeat("value", 0);
        }
    }
}

namespace WithUsingAlias
{
    using Seq = System.Linq.Enumerable;

    public class C
    {
        public void M()
        {
            _ = Seq.Repeat("value", 0);
        }
    }
}
