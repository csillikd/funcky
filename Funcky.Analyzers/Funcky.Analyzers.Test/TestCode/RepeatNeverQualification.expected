namespace WithUsingStatic
{
    using static System.Linq.Enumerable;

    public class C
    {
        public void M()
        {
            _ = Empty<string>();
        }
    }
}

namespace WithFullyQualifiedName
{
    public class C
    {
        public void M()
        {
            _ = System.Linq.Enumerable.Empty<string>();
        }
    }
}

namespace WithUsingAlias
{
    using Seq = System.Linq.Enumerable;

    public class C
    {
        public void M()
        {
            _ = Seq.Empty<string>();
        }
    }
}
